libcore/gof-file.c

```diff
@@ -1366,13 +1366,13 @@ gof_file_is_executable (GOFFile *file)
     {
         /* get the content type of the file */
         content_type = gof_file_get_ftype (file);
+        printf("Content type: %s\n", content_type);
```

Comment:
Please use g_debug () (if necessary; avoid excessive use).  Also space before bracket.

Reply:
Forgot to remove it before push. Sorry